.github/workflows/cypress.yml

```diff
@@ -102,7 +102,17 @@ jobs:
         run: npx @wordpress/env@latest start
 
       - name: Run Cypress Tests
-        run: npm run test:e2e
+        uses: replayio/action-cypress@v0.2.2
+        with:
+          api-key: ${{ secrets.RECORD_REPLAY_API_KEY }}
+          issue-number: ${{ github.event.pull_request.number }}
```

Comment:
`issue-number` isn't valid any longer. It was dropped in 0.2.0 when we replaced the inline PR comment code with a [GitHub app](https://github.com/apps/replay-io).